msvc/sdk-build.props

```diff
@@ -90,7 +92,7 @@
       <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
       <RuntimeLibrary Condition="'$(Configuration)'=='Debug'">MultiThreadedDebugDLL</RuntimeLibrary>
       <SystemHeaderDeps>true</SystemHeaderDeps>
-      <AdditionalOptions>-Werror -Wno-microsoft %(AdditionalOptions)</AdditionalOptions>
+      <AdditionalOptions>-Werror -Wno-microsoft -Wno-c++1z-extensions -Wno-ignored-attributes -Wno-deprecated-declarations %(AdditionalOptions)</AdditionalOptions>
```

Comment:
The problem is that the C++/WinRT headers generate these warnings. In a near-future release those headers will be updated not to produce those warnings, but that fix is only going to work for VC++ and not for clang. They do support clang, but it's a second-class citizen.

One possible solution is to disable these warnings only around the C++/WinRT header inclusions. That would be cumbersome, except they're all already wrapped in COMIncludes.h/COMIncludes_End.h, so maybe an easy fix is to disable and re-enable warnings in those headers.

---
In reply to: [109239222](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109239222) [](ancestors = 109239222)